src/__deprecated__/components/form/form-button/form-button.component.js

```diff
@@ -16,6 +16,7 @@ const propsForButton = (props) => {
   return ({
     save: {
       as: 'primary',
+      type: undefined,
```

Comment:
```suggestion
      type: 'submit',
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/submit

Reply:
Made the change locally as I had to update the snapshot. Thank you.